Modules/ExampleInverter/include/ExampleInverter.h

```diff
@@ -0,0 +1,47 @@
+#ifndef ExampleInverter_h
+#define ExampleInverter_h
+
+// The following header file is generated by CMake and thus it's located in
+// the build directory. It provides an export macro for classes and functions
+// that you want to be part of the public interface of your module.
+#include <MitkExampleInverterExports.h>
+
+#include <mitkImage.h>
+
+#include <itkInvertIntensityImageFilter.h>
+
+namespace captk {
+/** \class Example Inverter
+ *  \brief Workshop Example Inverter Module API
+ *  This is a simple module to show how modules can be invoked from plugins.
+ *  This class acts as a wrapper for the ITK filter.
+ */
+class MITKEXAMPLEINVERTER_EXPORT ExampleInverter
+{
+	/** \brief  This class is basically a filter wrapper, so
+		 * we COULD inherit from mitk::ImageToImageFilter.
+		 * While this would be better design, there is some code level overhead,
+		 * and that would distract from the point of this example.
+		 * So we'll keep it simpler (and hopefully clearer) by just using a
+		 * few familiar functions.
+		 */
+public:
+	ExampleInverter();
```

Comment:
Could you rename the ExampleInverter to ExampleImageInvertor so it is very specific and clear?